__tests__/app.test.js

```diff
@@ -91,4 +91,54 @@ describe("GET /api/reviews/:review_id", () => {
             expect(body.msg).toBe("Bad Request.")
         })
     })
+})
+
+describe("GET /api/reviews/:review_id/comments", () => {
+    test("200: returns an array of comments for a given review_id with specific properties", () => {
+        return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then(( { body } ) => {
+            const comments = body.comments
+            expect(comments).toBeInstanceOf(Array)
+            expect(comments).toHaveLength(3)
```

Comment:
Like that you're checking the array length to ensure you're not getting back any empty arrays before the loop!